plugins/modules/monitoring/uptimerobot.py

```diff
@@ -8,138 +8,178 @@
 
 
 DOCUMENTATION = '''
-
 module: uptimerobot
-short_description: Pause and start Uptime Robot monitoring
+short_description: Create, remove, pause and start monitors in Uptimerobot
```

Comment:
Is `Uptimerobot` the official spelling of the project? Or why did you change `Uptime Robot` to `Uptimerobot`?

Reply:
Changed in: https://github.com/ansible-collections/community.general/pull/961/commits/5d5c6bcc163c83624ae1294289a67fdc47fb9029